src/components/About.js

```diff
@@ -6,35 +6,31 @@ export default function About() {
     <section className="about-section  container">
       <div className="row mb-1">
         <div className="col-2 col-md-1 box-violet">
-          <h5 className="vertical-box-header about">ABOUT US</h5>
```

Comment:
This was done so that the nav links display correctly in the mobile design. If you notice, on mobile, we have About Us section at the bottom of the How To Help section. We can either leave it as it was before or if you have other suggestions to fix that issue, we can discuss.

Reply:
In Figma it looks like the About section should be below How to Help on both (all) size screens. On my local machine, the master branch is rendering in the opposite order.  Are you looking at the test site? I think I deployed there when I was testing this change, so it is showing the right order.

Please let me know if I am missing something.  It looks to me like these changes are actually needed.

Reply:
We can keep these changes, but we'll have to change the orders of the two sections. Maybe @wznd can take a look at it when he is fixing the styles.